### ISSUE TITLE
Cannot import rclone.config

### ISSUE THREAD

**Issue by caspertone2003** [kaczmarkiewiczp/rcloneExplorer#156]
Just installed last version in android 7.
Screen for config import pops but I am unable to navigate to select the config file I downloaded.
RcloneExplorer only searches in "recent" and "downloaded" but only shows media files and apks.
Is there a workaround? Could I copy the config file to the "expected" place so import is not necessary?
Thanks

**Comment by buywetwok** [comment]
1. Import
2. Tap on the three dots button on the very right
3. Tap "Show internal storage"
4. Tap on the Hamburger button on the very left 
4. Device should be visible as a source to browse and select rclone.config from any location.

![3dot](https://user-images.githubusercontent.com/39696077/42164492-4b9bf70e-7e5a-11e8-9b39-cc8137533566.png)

**Comment by caspertone2003** [comment]
@buywetwok
 
Thanks for handhelding me ... was prety obvious once you get into the mood... I feel so dumb!
was so easy but I did not got to imagine that I had to do right then left...

Now working sweetly. My android box is something different ;-)

Now I have to find out how to stream using it. My first trial with mega failed but I read in rclone forums that there are some issues with mega...

Thanks again!

CT
